mappings/net/minecraft/world/gen/feature/StructureFeature.mapping

```diff
@@ -86,12 +86,26 @@ CLASS net/minecraft/class_3195 net/minecraft/world/gen/feature/StructureFeature
 	METHOD method_28664 init ()V
 	METHOD method_28665 getCodec ()Lcom/mojang/serialization/Codec;
 	METHOD method_36420 getUndergroundWaterCreatureSpawns ()Lnet/minecraft/class_6012;
+	METHOD method_38428 getCornerInGroundHeight (Lnet/minecraft/class_2794;IIIILnet/minecraft/class_5539;)[I
+		ARG 0 generator
+		ARG 1 x
+		ARG 2 deltaX
+		ARG 3 z
+		ARG 4 deltaZ
+		ARG 5 world
 	METHOD method_38429 getLowestCornerInGroundHeight (Lnet/minecraft/class_2794;IILnet/minecraft/class_1923;Lnet/minecraft/class_5539;)I
 		ARG 0 generator
 		ARG 1 deltaX
 		ARG 2 deltaZ
 		ARG 3 chunkPos
 		ARG 4 world
+	METHOD method_38430 canGenerate (Lnet/minecraft/class_2794;Lnet/minecraft/class_5539;Ljava/util/function/Predicate;Lnet/minecraft/class_2902$class_2903;II)Z
```

Comment:
```suggestion
	METHOD method_38430 checkBiome (Lnet/minecraft/class_2794;Lnet/minecraft/class_5539;Ljava/util/function/Predicate;Lnet/minecraft/class_2902$class_2903;II)Z
```
This finds a starting biome to send to the predicate to check, called by chunk generators